airbyte/mcp/_local_ops.py

```diff
@@ -162,7 +167,7 @@ def get_source_stream_json_schema(
     ],
 ) -> dict[str, Any]:
     """List all properties for a specific stream in a source connector."""
-    source: Source = get_source(source_connector_name)
+    source: Source = get_source(source_connector_name, docker_image=is_docker_installed() or False)
```

Comment:
Make all of these multiline so they are easier to read. E.g.


        source = get_source(
            connector_name,
            docker_image=is_docker_installed() or False,
        )